features/image-transformations/image-enhancement-and-color-manipulation.md

```diff
@@ -133,3 +133,117 @@ URL - [https://ik.imagekit.io/demo/lipstick_2M.png](https://ik.imagekit.io/demo/
+
+![](https://ik.imagekit.io/demo/tr:h-300,e-gradient-ld-top_from-green_to-00FF0010_sp-1/sample_image.jpg)
+{% endtab %}
+{% endtabs %}
+
+#### Gradient as a background
+
+For input images containing a solid object over a transparent background, gradient can also be applied as an underlay.
+
+##### Usage syntax
+
+```markup
+https://ik.imagekit.io/<imagekitId>/your-image.png?tr=e-gradient-from-red_to-white:l-image,i-your-image.png,t-false,l-end
+```
+
+##### Examples
+
+{% tabs %}
+{% tab title="Original Image" %}
+URL - [https://ik.imagekit.io/demo/tr:w-500/lipstick_2M.png](https://ik.imagekit.io/demo/tr:w-500/lipstick_2M.png)
+
+![](https://ik.imagekit.io/demo/tr:w-500/lipstick_2M.png)
+{% endtab %}
+
+{% tab title="Gradient as a background" %}
+URL - [https://ik.imagekit.io/demo/tr:e-gradient-from-lightskyblue_to-mintcream:l-image,i-lipstick_2M.png,t-false,l-end:w-500/lipstick_2M.png](https://ik.imagekit.io/demo/tr:e-gradient-from-lightskyblue_to-mintcream:l-image,i-lipstick_2M.png,t-false,l-end:w-500/lipstick_2M.png)
+
+![](https://ik.imagekit.io/demo/tr:e-gradient-from-lightskyblue_to-mintcream:l-image,i-lipstick_2M.png,t-false,l-end:w-500/lipstick_2M.png)
+{% endtab %}
+{% endtabs %}
+
+{% hint style="info" %}
+If you experience a positional displacement between the original image and the overlay image, it would be worthwhile to use `t-false` to disable default trimming of the overlay layer like `l-image,i-your-image.png,t-false,l-end`.
+{% endhint %}
+
+#### Adding gradient blocks over images
+
+You can also add a gradient block over an input image.
+
+##### Usage syntax
+
+```markup
+https://ik.imagekit.io/<imagekitId>/your-image.png?tr=l-image,i-ik_canvas,e-gradient,l-end
+```
+
+You can also control the position of the gradient color overlay using these [positional parameters](./overlay-using-layers.md#position-of-layer).
+
+
+##### Transformation of gradient block overlay
+
+Following transformation parameters are supported on the gradient block overlay inside a layer.
+
+| Parameter                                                                                | Description                        |
+|------------------------------------------------------------------------------------------|------------------------------------|
+| w                                                                                        | Width of solid color block. It can also accept arithmetic expressions such as `bw_div_2`, or `bw_mul_0.8`. Learn more about arithmetic expressions [here](../arithmetic-expressions-in-transformations.md). |
+| h                                                                                        | Height of solid color block. It can also accept arithmetic expressions such as `bh_div_2`, or `bh_mul_0.8`. Learn more about arithmetic expressions [here](../arithmetic-expressions-in-transformations.md).  |
```

Comment:
```suggestion
| h                                                                                        | Height of the gradient block. It can also accept arithmetic expressions such as `bh_div_2`, or `bh_mul_0.8`. Learn more about arithmetic expressions [here](../arithmetic-expressions-in-transformations.md).  |
```